website/content/docs/job-specification/service.mdx

```diff
@@ -70,9 +70,9 @@ This section of the documentation only cover the job file fields and blocks
 for service discovery. For more details on using Nomad with Consul please see
 the [Consul integration documentation][service-discovery].
 
-Nomad 0.10 also allows specifying the `service` block at the task group level.
+The `service` block can also be at task group level.
 This enables services in the same task group to opt into [Consul
-Connect][connect] integration.
+Service Mesh][connect] integration.
```

Comment:
Did this get re-branded? I think we have "Connect" all over the place in the Nomad docs.

Reply:
Yep, it's "Consul Service Mesh" now. I know that Nomad mostly refers to it as Connect (including the job spec itself), but googling for "Nomad Consul Connect" results in those two high level docs first, both of which use "Service Mesh":

https://developer.hashicorp.com/nomad/docs/integrations/consul/service-mesh

https://developer.hashicorp.com/consul/docs/connect/nomad

So I think at least _some_ of the high level docs in Nomad land have started getting updated to the new wording, and opportunistically I updated this one too.